fh-ios-sdk/FHConfig.m

```diff
@@ -21,13 +21,18 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         NSString *path =
-            [[NSBundle bundleForClass:[self class]] pathForResource:@"fhconfig" ofType:@"plist"];
+            [[NSBundle mainBundle] pathForResource:@"fhconfig" ofType:@"plist"];
         if (path) {
             self.properties = [NSMutableDictionary dictionaryWithContentsOfFile:path];
         } else {
-            @throw([NSException exceptionWithName:@"fhconfigException"
-                                           reason:@"fhconfig.plist was not located"
-                                         userInfo:nil]);
+            path = [[NSBundle bundleForClass:[self class]] pathForResource:@"fhconfig" ofType:@"plist"];
```

Comment:
I thought maybe initially as you could have a "default" config which talks to your instance. A bit more thought and that seems dumb and unlikely. You need to spin up your app with an appropriate configuration and there is no "default" config which makes sense.